Documentation/En/Index.xyl

```diff
@@ -611,20 +612,20 @@ public function case_sum()
     <li><strong>Concurrent</strong>, “all” test cases at the same time in
     separated processes.</li>
   </ul>
-  <p>When running tests, isolation is really important: No memory conflict, no
-  execution conflict, state of the System Under Test is reset each time etc.
+  <p>When running tests, isolation is really important: No memory conflicts, no
+  execution conflicts, the state of the System Under Test is reset each time etc.
```

Comment:
Because there could be many classes of memory or execution conflicts. We are not talking about a specific instance of a memory conflict, but memory conflicts in general. I would write it as I have edited it, but whatever makes more sense to you.